docs/source/writeups/doppler.rst

```diff
@@ -143,6 +145,106 @@ Internally, rydiqule defines the necessary components for Doppler averaging via
 This construction has the benefit of allowing for meshes (ie velocity classes) to be defined in a general way relative to the distribution width :math:`v_P`,
 making them easily re-usable for any velocity distribution that obeys the Maxwell-Boltzmann distribution.
 
+Analytically Averaging Velocity Classes
+---------------------------------------
+
+Rydiqule's implementation of analytic doppler-averaging follows the propagator method derived in `Exact steady state of perturbed open quantum systems <https://arxiv.org/abs/2501.06134>`_
+by Omar Nagib and Thad Walker. 
+In one spatial dimension, the time evolution of a system is governed by the master equation in the superoperator form
+
+.. math::
+
+    \dot{\rho} = \mathcal{L} \rho 
+
+At steady state, this equation becomes
+
+.. math::
+
+    \mathcal{L} \rho = 0
+
+Considering a velocity class, :math:`v`, :math:`\mathcal{L}` can be divided into two parts:
+
+.. math::
+
+    \mathcal{L} \rho_v = (\mathcal{L_0} + v \mathcal{L_1}) \rho_v = 0
+
+where :math:`\mathcal{L_0}` and :math:`\mathcal{L_1}` do not depend on :math:`v`.
+A propagator, :math:`G_v`, is then constructed such that
+
+.. math::
+
+    G_v \rho_0 = \rho_v
+
+where :math:`\rho_0` is the unique steady state solution when :math:`v=0`. As shown by Nagib and Walker, this propagator :math:`G_v` is constructed as
+
+.. math::
+
+    G_v = \frac{\mathbb{1}}{\mathbb{1} + v \mathcal{L}_0^- \mathcal{L}_1}
+
+where :math:`\mathcal{L}_0^-` is the Drazin inverse of :math:`\mathcal{L}_0`.
+Suppose that :math:`\mathcal{L}_0^- \mathcal{L}_1` can be eigendecomposed as
+
+.. math::
+
+    \mathcal{L}_0^- \mathcal{L}_1 = \sum_{\lambda = \lambda_1}^{\lambda_N} \lambda r_{\lambda} l_{\lambda}^T 
+
+where :math:`r_{\lambda}` and :math:`l_{\lambda}` are the right and left eigenvectors of :math:`\mathcal{L}_0^- \mathcal{L}_1`, respectively.
+Then the propagator is decomposed as
+
+.. math::
+
+    G_v = \sum_{\lambda = \lambda_1}^{\lambda_N} \frac{1}{1 + v \lambda} r_{\lambda} l_{\lambda}^T 
+
+Thus,
+
+.. math::
+
+    \rho_v = G_v \rho_0 = \sum_{\lambda = \lambda_1}^{\lambda_N} \frac{1}{1 + v \lambda} r_{\lambda} l_{\lambda}^T \rho_0
+
+Then the ensemble average can be computed by integrating over :math:`v`:
+
+.. math::
+
+    \bar{\rho} = \int \rho_v f_{v} dv = \sum_{\lambda = 0} r_{\lambda} l_{\lambda}^T \rho_0 
+    + \sum_{\lambda \not= 0} \frac{\sqrt{\pi/2}}{\sqrt{-\lambda^2} \sigma_v} \exp{\frac{-1}{2 \lambda^2 \sigma_v^2}} 
+    \left(1 + \text{erf}\left[ \frac{\sqrt{-\lambda^2}}{\sqrt{2} \lambda^2 \sigma_v} \right] \right) r_{\lambda} l_{\lambda}^T \rho_0
```

Comment:
We should update this to `erfc` and mention we use `erfcx` for stability under the hood. Probably should link to scipy.erfcx when doing so.